runtime/vm/module.xml

```diff
@@ -160,6 +160,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<library name="j9stackmap"/>
 			<library name="j9hookable"/>
 			<library name="j9omr" type="external"/>
+			<library name="omrglue" type="external">
```

Comment:
Can you explain why this is necessary?  I don't understand why the vm would need to grow this dependency.

Reply:
Removed